src/rviz_visual_tools.cpp

```diff
@@ -629,6 +641,187 @@ bool RvizVisualTools::publishMarkers(const visualization_msgs::MarkerArray &mark
   return true;
 }
 
+bool RvizVisualTools::publishCone(const Eigen::Affine3d &pose, double angle, const rviz_visual_tools::colors &color, double scale)
+{
+  return publishCone(convertPose(pose), angle, color, scale);
+}
+
+bool RvizVisualTools::publishCone(const geometry_msgs::Pose &pose, double angle, const rviz_visual_tools::colors &color, double scale)
+{
+  triangle_marker_.header.stamp = ros::Time::now();
+  triangle_marker_.id++;
+
+  triangle_marker_.color = getColor(color);
+  triangle_marker_.pose = pose;
+  
+  geometry_msgs::Point p[3];
+  double d_theta = M_PI / 16.0;
```

Comment:
Also I would prefer it spelled out ``delta_theta`` to help others. I've fixed both already.